src/data/hashMap/PLHashMap.spec.ts

```diff
@@ -20,7 +20,10 @@ describe('stdlib/data/PLHashMap', () => {
     it('should return with the JS representation', () => {
       const js = plHashMap(plString('a'), plNumber(1), plString('b'), plNumber(2)).toJS()
       expect(Array.from(js.keys())).toEqual(['a', 'b'])
-      expect(Array.from(js.values())).toEqual([1, 2])
+      expect(Array.from(js.values())).toEqual([
```

Comment:
@vektor8891 did you pull the latest master branch?
In the latest version PLNumber `toJS` will return with a single number and only the `toJSON` will return wit the decimal representation.

Reply:
@maxinteger you are right, the code wasn't updated.